js/src/pages/courses/Section.js

```diff
@@ -9,6 +9,7 @@ import CardActionArea from '@material-ui/core/CardActionArea';
 import CardActions from '@material-ui/core/CardActions';
 import Typography from '@material-ui/core/Typography';
 import makeStyles from '@material-ui/core/styles/makeStyles';
+import LinearProgress from "@material-ui/core/LinearProgress";
```

Comment:
Did you run `npm run format` and `npm run lint-fix` before pushing? It should have fixed the quotes used - ' instead of "

Reply:
I was not aware of those.
Now I uploaded the automatic fixes.